src/main/java/com/example/project_spring_boot/controller/ItemController.java

```diff
@@ -29,9 +31,19 @@ public ResponseEntity <List<Item>> getItems() {
         return new ResponseEntity<>(itemService.getItems(), HttpStatus.OK);
     }
 
+    @GetMapping("/item/owner/{id}")
+    public ResponseEntity <List<Item>> getItemsByUserId(@PathVariable Long id) {
+        return new ResponseEntity<>(itemService.getItemsByUserId(id), HttpStatus.OK);
+    }
+
      @PostMapping("/item")
      public ResponseEntity<Item> saveItem(@RequestBody Item item) {
         return new ResponseEntity<>(itemService.createItem(item), HttpStatus.CREATED);
      }
+
+     @PutMapping("/item/{itemId}")
```

Comment:
I know we talked briefly about put and patch before, shouldn´t it be considered to be a partial change to item? and therefore be patch?
![image](https://user-images.githubusercontent.com/89969770/207024689-439e5432-6c80-420b-92af-6041e8fdff2c.png)

Reply:
You are right, as usual ;) I will change to patch right away :)